app/views/annotator/index.html.haml

```diff
@@ -3,7 +3,7 @@
   .annotator-page-subcontainer
     .annotator-page-title
       .text
-        = @page_name
```

Comment:
Here it needs to use that variable, @Bilelkihal will explain.

Reply:
@muhammedBkf 
Yes, we use this dynamic name because we have 3 versions of the annotator:
annotator, annotator plus, ncbo_annotator_plus.
Since they are identical in the UI side then we tried to use only one html page and this variable is used to display the name of the annotator, so you can't remove it, you can just keep it as it is "Annotator" it's not a big deal